collection/sites/pastebin.yml

```
@@ -0,0 +1,10 @@
+name: Pastebin
+host: pastebin.com
+example: https://pastebin.com/9jEf44nc
+extract: "#pastebin\\.com/(?!u/)(?:\\w+(?:\\.php\\?i=|/))?(?'id'\\w+)#"
```

Related to the comment below.

```suggestion
extract: "#pastebin\\.com/(?!u/)(?:\\w+(?:\\.php\\?i=|/))?(?'id'\\w+)(?'dark'\\?theme=dark)?#"
```

---

Or just -dark or /dark

---

> Or just -dark or /dark

Neither of them worked for me directly in the link. How it's supposed to be used?

---

Actually I don't think user's should be able to do this...how would they even know? If it could be style dependent (like a style adds something that would trigger the dark mode) that could be an option, otherwise just use default appearance (even though I prefer dark modes)

---

Well, the option is documented on pastebin.com, so can be used by someone [advanced users] who is aware, others can use pastebin as is :)